package.json

```diff
@@ -50,10 +50,9 @@
     "@storybook/react-vite": "^7.6.17",
     "@swc-node/register": "^1.6.6",
     "@swc/core": "^1.3.68",
-    "@testing-library/dom": "^8.11.0",
```

Comment:
`jest-dom` already depends on this library and we never use it directly. So this avoids version mismatches (which I did hit while working on this PR) when we bump testing library versions.